reana-ui/src/components/Search.js

```diff
@@ -2,20 +2,20 @@
   -*- coding: utf-8 -*-
 
   This file is part of REANA.
-  Copyright (C) 2020 CERN.
+  Copyright (C) 2020, 2021, 2022 CERN.
```

Comment:
Very minor: copyright year was not updated in most of the other touched files

Reply:
Updated, thanks!

Did you manage to find a nice way to keep this updated in VSCode?

Reply:
Not yet, is seems there are no extensions which would do exactly this task in VSCode unfortunately. However there are extensions like 'Run on Save' which could trigger custom task like this after saving a file, but I haven't tried to implement it.